### PR TITLE
Volunteering section modified

### PR DESCRIPTION
**Pull request by raksha009**
Made the following changes:
1. Added new images in the experience section folder itself
2. Added new animations
3. used cardv for volunteering section as card class was already used in mentorship section
4. Added comments in CSS file 

Issue #80

### CODE REVIEW

**Comment by raksha009** [comment]
ezgif com-video-to-gif (2)

**Comment by smaranjitghose** [comment]
Volunteering is something, we usually do out of passion. So, having a dedicated view certificate link to prove it seems odd
@anushbhatia will suggest you changes for the animation

File: experience.html [hunk 1]
```diff
@@ -144,9 +145,58 @@ <h6 class="text-muted text-center py-2 heading2">I have been contributing variou
...
+<!--    <main class="page-content volunteership">-->
       <!-- Dynamic volunteership Card -->
-    </main>
+<!--    </main>-->
+    <br> <br>
+      <div class="wrapper">
+        <div class="cardv" style="margin-left: 90px"><img src="assets/images/experience-page/gssoc.png"/>
+          <div class="info">
+            <h1>GirlScript Summer Of Code 2020</h1>
+            <p>Responsible for handling the projects GirlScript App and GirlScript Website Boilerplate.</p>
```
Comment (smaranjitghose) [inline experience.html:156]: Isn't this  supposed to be loaded from a script?

File: experience.html [hunk 2]
```diff
@@ -198,7 +248,7 @@ <h5 class="card-title">HakinCode</h5>
     <link rel="stylesheet" href="assets\css\style.css" />
     <link rel="stylesheet" href="assets\css\experience.css" />
     <script src="assets/js/main.js"></script>
-    <script src="assets/js/experience.js"></script>
+<!--    <script src="assets/js/experience.js"></script>-->
```
Comment (smaranjitghose) [inline experience.html:251]: ???

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by raksha009** [comment]
Okay, I'll remove that view certificate option.
Also, I commented out the Js part cause the images weren't loading up (or showing ) in pc. So, the same issue persisted even when I tried uploading images via css too. 
Moreover, I wasn't able to change animation when the text was uploading from JS part. So, hence tried putting all display part in html.

**Comment by raksha009** [comment]
@smaranjitghose @anushbhatia plz suggest

**Comment by anushbhatia** [comment]
Don't upload images with the help of the CSS use HTML for that. See use main.js where you store the data for different images and the text. In the HTML just load the data form that file so that if we add a new volunteering experience it automatically gets added.

**Comment by shreyakapoor08** [comment]
@raksha009 Resolve the merge conflict

**Pull request closed without merging**